manifests/install/container_runtime.pp

```diff
@@ -87,6 +87,7 @@
   }
 
   if $manage_repo {
+    include k8s::repo
```

Comment:
when you make this a `require`, you can drop `Class['k8s::repo'] -> Package['k8s container manager']`
```suggestion
    require k8s::repo
```